src/components/CookieWarning.js

```diff
@@ -1,28 +1,24 @@
 import _ from 'lodash/fp'
-import { useState } from 'react'
 import { div, h } from 'react-hyperscript-helpers'
 import { ButtonPrimary, ButtonSecondary, Link } from 'src/components/common'
 import { Ajax } from 'src/libs/ajax'
 import { signOut } from 'src/libs/auth'
 import colors from 'src/libs/colors'
 import * as Nav from 'src/libs/nav'
-import { getLocalPref, setLocalPref } from 'src/libs/prefs'
 import { authStore } from 'src/libs/state'
 import * as Utils from 'src/libs/utils'
 
 
 export const cookiesAcceptedKey = 'cookiesAccepted'
 
-export const CookieWarning = () => {
+const CookieWarning = () => {
   const signal = Utils.useCancellation()
-  const [acceptedCookies, setAcceptedCookies] = useState(getLocalPref(cookiesAcceptedKey))
-  authStore.subscribe(state => {
-    setAcceptedCookies(getLocalPref(cookiesAcceptedKey))
-  })
+  const { cookiesAccepted } = Utils.useStore(authStore)
+
   const acceptCookies = acceptedCookies => {
-    setLocalPref(cookiesAcceptedKey, acceptedCookies)
-    setAcceptedCookies(getLocalPref(cookiesAcceptedKey))
+    authStore.update(_.set('cookiesAccepted', acceptedCookies))
```

Comment:
where would I be able to see this? I don't see it appearing in local storage at all (I checked each localhost:3000 option). Actually, even if I hit accept and refresh the page, I get the banner again. 
![cookies tab after hitting accept](https://user-images.githubusercontent.com/5375000/106306294-dc0eb800-622b-11eb-9d9a-9cfec6291b79.png)

Reply:
So, you could see it in the Local Storage section on that area you are in. You're currently looking at cookies specifically. However, we don't save it for users who are signed out, because we can't verify that a user who comes back is the same. (E.g., going to the site on a library computer)

Reply:
I do see it now! It also seems to work as expected for a second user who has not yet accepted cookies (userA signs in, accepts, signs out, userB signs in and should see the cookies banner). I also tried logging in and out for both users.